docs/mathematics/prime-number.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Prime Number
+slug: /mathematics/prime-number
+---
+
+## Introduction
+
+Prime numbers are the positive integers having only two factors, 1 and the integer itself.
+
+## Explanation
+
+If `p` is a prime, then it’s only factors are necessarily 1 and p itself. Any number which does not follow this is termed as composite numbers, which means that they can be factored into other positive integers.
+<a href="https://thirdspacelearning.com/wp-content/uploads/2019/11/How-do-prime-numbers-work.png/></a>
```

Comment:
I guess this is breaking the build here. Please remove it and use only your own images

Reply:
@HarshCasper my image means, should I download and then use?